benchmark/build.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/zsh
+
+ROOT=$PWD/..
+rm -rf runner/out runner/processor-1.0-SNAPSHOT.jar runner/com tachiyomi runner/lib
+./gradlew :jar
+cd $ROOT/benchmark/exhaustive-processor
+./gradlew build
+cp processor/build/libs/processor-1.0-SNAPSHOT.jar ../runner/
+cd $ROOT/benchmark
+git clone https://github.com/inorichi/tachiyomi.git
+cd tachiyomi
+git checkout 938339690eecdfe309d83264b6a89aff3c767687
+git apply $ROOT/benchmark/runner/tachi.patch
+./gradlew :app:copyDepsDevDebug
+mkdir $ROOT/benchmark/runner/lib && cp app/build/output/devDebug/lib/* $ROOT/benchmark/runner/lib/
+cd $ROOT
+./gradlew -PkspVersion=2.0.255 clean publishAllPublicationsToTestRepository
+cp -a build/repos/test/. $ROOT/benchmark/runner
+cd $ROOT/benchmark/runner
+./runner.sh -cp $(echo lib/*.jar | tr ' ' ':') $ROOT/benchmark/tachiyomi/app/**/*.kt
```

Comment:
Is the `**/` here a zsh thing? If so, you can replace it with something like:

`find $ROOT/benchmark/tachiyomi/app/ -name "*.kt" | xargs ./runner.sh -cp $(echo lib/*.jar | tr ' ' ':') `

Reply:
Maybe remove the run of the benchmark from the builder script. That way, if the build succeeds, but the processor fails, golem will correctly report it as a benchmark run failure and not as a failure to build KSP.